packages/gdk_base.rb

```diff
@@ -18,19 +18,27 @@ def self.install
 
       # Do not edit this file. It will be overwritten by updates.
 
-      SCALE=1
-      [ -z "$DISPLAY" ] && DISPLAY=:0
-      [ -z "$GDK_BACKEND" ] && GDK_BACKEND=x11
-      RESOLUTION=$(xdpyinfo | awk '/dimensions:/ { print $2 }' | cut -d'x' -f1)
-      [[ $RESOLUTION -gt 1500 && $RESOLUTION -lt 2500 ]] && SCALE=1.5
-      [[ $RESOLUTION -ge 2500 && $RESOLUTION -lt 3500 ]] && SCALE=2
-      [[ $RESOLUTION -ge 3500 && $RESOLUTION -lt 4500 ]] && SCALE=2.5
-      [[ $RESOLUTION -ge 4500 && $RESOLUTION -lt 5500 ]] && SCALE=3
-      [[ $RESOLUTION -gt 5500 ]] && SCALE=3.5
-      export GDK_BACKEND=$GDK_BACKEND
-      export GDK_SCALE=$SCALE
-      export QT_SCALE_FACTOR=$SCALE
-      export DISPLAY=$DISPLAY
+      GDK_BACKEND=${GDK_BACKEND:-x11}
+      function roundhalves {
+                    echo "$1 * 2" | bc | xargs -I@ printf "%1.f" @ | xargs -I% echo "% * .5" | bc
+            }
+      pxwidth=$(WAYLAND_DISPLAY=wayland-0 wayland-info -i wl_output | grep width: | grep px | head -n 1 | awk '{print $2}')
+      lwidth=$(WAYLAND_DISPLAY=wayland-0 wayland-info -i zxdg_output_manager_v1 | grep logical_width:  | sed 's/,//' | awk '{print $2}')
+      # echo "pxwidth: $pxwidth, lwidth: $lwidth"
```

Comment:
What is wrong with leaving this uncommented?

Reply:
It's debug code, so why print out?

Reply:
I guess it would help to indicate the file was sourced?  But maybe it would make sense to echo the `*SCALE*` variables instead?

Reply:
Done.